# Before your start:
- Read the README.md file
- Comment as much as you can and use the resources (README.md file)
- Happy learning!

In [11]:
import numpy as np
impo
rt pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns
import scipy.stats as stats 

# Challenge 1 - Exploring the Data

In this challenge, we will examine all salaries of employees of the City of Chicago. We will start by loading the dataset and examining its contents

In [2]:
# Run this code:

salaries = pd.read_csv('../Current_Employee_Names__Salaries__and_Position_Titles.csv')

Examine the `salaries` dataset using the `head` function below.

In [3]:
# Your code here:
salaries.head(10)

Name                              Job Titles  \
0      AARON,  JEFFERY M                                SERGEANT   
1        AARON,  KARINA   POLICE OFFICER (ASSIGNED AS DETECTIVE)   
2    AARON,  KIMBERLEI R                CHIEF CONTRACT EXPEDITER   
3    ABAD JR,  VICENTE M                       CIVIL ENGINEER IV   
4      ABASCAL,  REECE E             TRAFFIC CONTROL AIDE-HOURLY   
5  ABBASI,  CHRISTOPHER               STAFF ASST TO THE ALDERMAN   
6  ABBATACOLA,  ROBERT J                     ELECTRICAL MECHANIC   
7     ABBATE,  JOSEPH L                  POOL MOTOR TRUCK DRIVER   
8  ABBATEMARCO,  JAMES J                       FIRE ENGINEER-EMT   
9       ABBATE,  TERRY M                          POLICE OFFICER   

         Department Full or Part-Time Salary or Hourly  Typical Hours  \
0            POLICE                 F           Salary            NaN   
1            POLICE                 F           Salary            NaN   
2  GENERAL SERVICES                 F           Salary            NaN   
3       WATER MGMNT                 F           Salary            NaN   
4              OEMC                 P           Hourly           20.0   
5      CITY COUNCIL                 F           Salary            NaN   
6          AVIATION                 F           Hourly           40.0   
7     STREETS & SAN                 F           Hourly           40.0   
8              FIRE                 F           Salary            NaN   
9            POLICE                 F           Salary            NaN   

   Annual Salary  Hourly Rate  
0       101442.0          NaN  
1        94122.0          NaN  
2       101592.0          NaN  
3       110064.0          NaN  
4            NaN        19.86  
5        50436.0          NaN  
6            NaN        46.10  
7            NaN        35.60  
8       103350.0          NaN  
9        93354.0          NaN

We see from looking at the `head` function that there is quite a bit of missing data. Let's examine how much missing data is in each column. Produce this output in the cell below

In [4]:
# Your code here:
salaries.isnull().sum().sort_values(ascending=False).head(10)

Hourly Rate          25161
Typical Hours        25161
Annual Salary         8022
Salary or Hourly         0
Full or Part-Time        0
Department               0
Job Titles               0
Name                     0
dtype: int64

In [5]:
salaries.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 33183 entries, 0 to 33182
Data columns (total 8 columns):
 #   Column             Non-Null Count  Dtype  
---  ------             --------------  -----  
 0   Name               33183 non-null  object 
 1   Job Titles         33183 non-null  object 
 2   Department         33183 non-null  object 
 3   Full or Part-Time  33183 non-null  object 
 4   Salary or Hourly   33183 non-null  object 
 5   Typical Hours      8022 non-null   float64
 6   Annual Salary      25161 non-null  float64
 7   Hourly Rate        8022 non-null   float64
dtypes: float64(3), object(5)
memory usage: 2.0+ MB


Let's also look at the count of hourly vs. salaried employees. Write the code in the cell below

In [6]:
# Your code here:
salaries['Salary or Hourly'].value_counts()

Salary    25161
Hourly     8022
Name: Salary or Hourly, dtype: int64

What this information indicates is that the table contains information about two types of employees - salaried and hourly. Some columns apply only to one type of employee while other columns only apply to another kind. This is why there are so many missing values. Therefore, we will not do anything to handle the missing values.

There are different departments in the city. List all departments and the count of employees in each department.

In [7]:
# Your code here:

salaries["Department"].value_counts()

POLICE                   13414
FIRE                      4641
STREETS & SAN             2198
OEMC                      2102
WATER MGMNT               1879
AVIATION                  1629
TRANSPORTN                1140
PUBLIC LIBRARY            1015
GENERAL SERVICES           980
FAMILY & SUPPORT           615
FINANCE                    560
HEALTH                     488
CITY COUNCIL               411
LAW                        407
BUILDINGS                  269
COMMUNITY DEVELOPMENT      207
BUSINESS AFFAIRS           171
COPA                       116
BOARD OF ELECTION          107
DoIT                        99
PROCUREMENT                 92
INSPECTOR GEN               87
MAYOR'S OFFICE              85
CITY CLERK                  84
ANIMAL CONTRL               81
HUMAN RESOURCES             79
CULTURAL AFFAIRS            65
BUDGET & MGMT               46
ADMIN HEARNG                39
DISABILITIES                28
TREASURER                   22
HUMAN RELATIONS             16
BOARD OF

# Challenge 2 - Hypothesis Tests

In this section of the lab, we will test whether the hourly wage of all hourly workers is significantly different from $30/hr. Import the correct one sample test function from scipy and perform the hypothesis test for a 95% two sided confidence interval.

In [9]:

checkPValue = lambda p,threshold=0.05: "--> Null(H0) hypotesis rejected" if p < threshold else "--> We cannot reject the null hypotesis"

In [13]:
salaries[salaries["Hourly Rate"].notnull()]

Name                      Job Titles  \
4           ABASCAL,  REECE E     TRAFFIC CONTROL AIDE-HOURLY   
6       ABBATACOLA,  ROBERT J             ELECTRICAL MECHANIC   
7          ABBATE,  JOSEPH L          POOL MOTOR TRUCK DRIVER   
10           ABBOTT,  BETTY L              FOSTER GRANDPARENT   
18       ABDULLAH,  LAKENYA N                  CROSSING GUARD   
...                       ...                             ...   
33164        ZUREK,  FRANCIS              ELECTRICAL MECHANIC   
33168  ZWARYCZ MANN,  IRENE A                  CROSSING GUARD   
33169      ZWARYCZ,  THOMAS J         POOL MOTOR TRUCK DRIVER   
33174        ZYGADLO,  JOHN P          MACHINIST (AUTOMOTIVE)   
33175     ZYGADLO,  MICHAEL J  FRM OF MACHINISTS - AUTOMOTIVE   

             Department Full or Part-Time Salary or Hourly  Typical Hours  \
4                  OEMC                 P           Hourly           20.0   
6              AVIATION                 F           Hourly           40.0   
7         STREETS & SAN                 F           Hourly           40.0   
10     FAMILY & SUPPORT                 P           Hourly           20.0   
18                 OEMC                 P           Hourly           20.0   
...                 ...               ...              ...            ...   
33164              OEMC                 F           Hourly           40.0   
33168              OEMC                 P           Hourly           20.0   
33169       WATER MGMNT                 F           Hourly           40.0   
33174  GENERAL SERVICES                 F           Hourly           40.0   
33175  GENERAL SERVICES                 F           Hourly           40.0   

       Annual Salary  Hourly Rate  
4                NaN        19.86  
6                NaN        46.10  
7                NaN        35.60  
10               NaN         2.65  
18               NaN        17.68  
...              ...          ...  
33164            NaN        46.10  
33168            NaN        17.68  
33169            NaN        35.60  
33174            NaN        46.35  
33175            NaN        48.85  

[8022 rows x 8 columns]

In [15]:
resA = stats.ttest_1samp(salaries[salaries["Hourly Rate"].notnull()]["Hourly Rate"], 30)
print(resA)
print(checkPValue(resA.pvalue))

Ttest_1sampResult(statistic=20.6198057854942, pvalue=4.3230240486229894e-92)
--> Null(H0) hypotesis rejected


We are also curious about salaries in the police force. The chief of police in Chicago claimed in a press briefing that salaries this year are higher than last year's mean of $86000/year a year for all salaried employees. Test this one sided hypothesis using a 95% confidence interval.

Hint: A one tailed test has a p-value that is half of the two tailed p-value. If our hypothesis is greater than, then to reject, the test statistic must also be positive.

In [16]:
# Your code here:
salaries.loc[salaries["Department"] == "POLICE"][salaries["Annual Salary"].notnull()]

/home/usuario/.local/lib/python3.6/site-packages/ipykernel_launcher.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.
  


Name                              Job Titles  \
0           AARON,  JEFFERY M                                SERGEANT   
1             AARON,  KARINA   POLICE OFFICER (ASSIGNED AS DETECTIVE)   
9            ABBATE,  TERRY M                          POLICE OFFICER   
11           ABDALLAH,  ZAID                           POLICE OFFICER   
12     ABDELHADI,  ABDALMAHD                           POLICE OFFICER   
...                       ...                                     ...   
33177        ZYGMUNT,  DAWID                           POLICE OFFICER   
33178   ZYLINSKA,  KATARZYNA                           POLICE OFFICER   
33179      ZYMANTAS,  LAURA C                          POLICE OFFICER   
33180       ZYMANTAS,  MARK E                          POLICE OFFICER   
33181     ZYRKOWSKI,  CARLO E                          POLICE OFFICER   

      Department Full or Part-Time Salary or Hourly  Typical Hours  \
0         POLICE                 F           Salary            NaN   
1         POLICE                 F           Salary            NaN   
9         POLICE                 F           Salary            NaN   
11        POLICE                 F           Salary            NaN   
12        POLICE                 F           Salary            NaN   
...          ...               ...              ...            ...   
33177     POLICE                 F           Salary            NaN   
33178     POLICE                 F           Salary            NaN   
33179     POLICE                 F           Salary            NaN   
33180     POLICE                 F           Salary            NaN   
33181     POLICE                 F           Salary            NaN   

       Annual Salary  Hourly Rate  
0           101442.0          NaN  
1            94122.0          NaN  
9            93354.0          NaN  
11           84054.0          NaN  
12           87006.0          NaN  
...              ...          ...  
33177        72510.0          NaN  
33178        72510.0          NaN  
33179        48078.0          NaN  
33180        90024.0          NaN  
33181        93354.0          NaN  

[13404 rows x 8 columns]

In [17]:

resB = stats.ttest_1samp(salaries.loc[salaries["Department"] == "POLICE"][salaries["Annual Salary"].notnull()]["Annual Salary"], 86000)
print(resB)
print(checkPValue(resB.pvalue))

Ttest_1sampResult(statistic=3.081997005712994, pvalue=0.0020603403550965137)
--> Null(H0) hypotesis rejected


/home/usuario/.local/lib/python3.6/site-packages/ipykernel_launcher.py:1: UserWarning: Boolean Series key will be reindexed to match DataFrame index.
  """Entry point for launching an IPython kernel.


Using the `crosstab` function, find the department that has the most hourly workers. 

In [18]:
# Your code here:
pd.crosstab(salaries["Department"], salaries["Salary or Hourly"])["Hourly"].sort_values(ascending=False)

Department
STREETS & SAN            1862
WATER MGMNT              1513
OEMC                     1273
AVIATION                 1082
GENERAL SERVICES          765
TRANSPORTN                725
PUBLIC LIBRARY            299
FAMILY & SUPPORT          287
CITY COUNCIL               64
FINANCE                    44
LAW                        40
ANIMAL CONTRL              19
POLICE                     10
MAYOR'S OFFICE              8
CULTURAL AFFAIRS            7
BUSINESS AFFAIRS            7
HUMAN RESOURCES             4
COMMUNITY DEVELOPMENT       4
HEALTH                      3
BUDGET & MGMT               2
FIRE                        2
PROCUREMENT                 2
DISABILITIES                0
DoIT                        0
COPA                        0
TREASURER                   0
HUMAN RELATIONS             0
CITY CLERK                  0
INSPECTOR GEN               0
BUILDINGS                   0
LICENSE APPL COMM           0
BOARD OF ETHICS             0
BOARD OF ELECTION           0

The workers from the department with the most hourly workers have complained that their hourly wage is less than $35/hour. Using a one sample t-test, test this one-sided hypothesis at the 95% confidence level.

In [19]:
# Your code here:
resC = stats.ttest_1samp(salaries.loc[salaries["Department"] == "STREETS & SAN"][salaries["Hourly Rate"].notnull()]["Hourly Rate"], 35)
print(resC)
print(checkPValue(resC.pvalue))


Ttest_1sampResult(statistic=-9.567447887848152, pvalue=3.3378530564707717e-21)
--> Null(H0) hypotesis rejected


/home/usuario/.local/lib/python3.6/site-packages/ipykernel_launcher.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.
  


# Challenge 3 - Constructing Confidence Intervals

While testing our hypothesis is a great way to gather empirical evidence for accepting or rejecting the hypothesis, another way to gather evidence is by creating a confidence interval. A confidence interval gives us information about the true mean of the population. So for a 95% confidence interval, we are 95% sure that the mean of the population is within the confidence interval. 
).

To read more about confidence intervals, click [here](https://en.wikipedia.org/wiki/Confidence_interval).


In the cell below, we will construct a 95% confidence interval for the mean hourly wage of all hourly workers. 

The confidence interval is computed in SciPy using the `t.interval` function. You can read more about this function [here](https://docs.scipy.org/doc/scipy-0.14.0/reference/generated/scipy.stats.t.html).

To compute the confidence interval of the hourly wage, use the 0.95 for the confidence level, number of rows - 1 for degrees of freedom, the mean of the sample for the location parameter and the standard error for the scale. The standard error can be computed using [this](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.sem.html) function in SciPy.

In [21]:
# Your code here:
mean = salaries[salaries["Hourly Rate"].notnull()]["Hourly Rate"].mean()

In [22]:
scale=stats.sem(salaries[salaries["Hourly Rate"].notnull()]["Hourly Rate"])
scale

0.1352368565101596

In [23]:
df=len(salaries[salaries["Hourly Rate"].notnull()]["Hourly Rate"])-1

In [24]:

stats.t.interval(0.95, df, loc=mean, scale=scale)

(32.52345834488425, 33.05365708767623)

Now construct the 95% confidence interval for all salaried employeed in the police in the cell below.

In [25]:
# Your code here:
mean = salaries[salaries["Department"] == "POLICE"][salaries["Annual Salary"].notnull()]["Annual Salary"].mean()


/home/usuario/.local/lib/python3.6/site-packages/ipykernel_launcher.py:2: UserWarning: Boolean Series key will be reindexed to match DataFrame index.
  


In [26]:
scale=stats.sem(salaries[salaries["Department"] == "POLICE"][salaries["Annual Salary"].notnull()]["Annual Salary"])
scale

/home/usuario/.local/lib/python3.6/site-packages/ipykernel_launcher.py:1: UserWarning: Boolean Series key will be reindexed to match DataFrame index.
  """Entry point for launching an IPython kernel.


157.82445675052244

In [27]:
df=len(salaries[salaries["Department"] == "POLICE"][salaries["Annual Salary"].notnull()])-1

/home/usuario/.local/lib/python3.6/site-packages/ipykernel_launcher.py:1: UserWarning: Boolean Series key will be reindexed to match DataFrame index.
  """Entry point for launching an IPython kernel.


In [28]:

stats.t.interval(0.95, df, loc=mean, scale=scale)

(86177.05631531784, 86795.77269094894)

# Challenge 4 - Hypothesis Tests of Proportions

Another type of one sample test is a hypothesis test of proportions. In this test, we examine whether the proportion of a group in our sample is significantly different than a fraction. 

You can read more about one sample proportion tests [here](http://sphweb.bumc.bu.edu/otlt/MPH-Modules/BS/SAS/SAS6-CategoricalData/SAS6-CategoricalData2.html).

In the cell below, use the `proportions_ztest` function from `statsmodels` to perform a hypothesis test that will determine whether the number of hourly workers in the City of Chicago is significantly different from 25% at the 95% confidence level.

In [29]:
# Your code here:
len(salaries[salaries["Salary or Hourly"]=="Hourly"]) / len(salaries["Salary or Hourly"])

0.24175029382515142

In [30]:
salaries

Name                              Job Titles  \
0          AARON,  JEFFERY M                                SERGEANT   
1            AARON,  KARINA   POLICE OFFICER (ASSIGNED AS DETECTIVE)   
2        AARON,  KIMBERLEI R                CHIEF CONTRACT EXPEDITER   
3        ABAD JR,  VICENTE M                       CIVIL ENGINEER IV   
4          ABASCAL,  REECE E             TRAFFIC CONTROL AIDE-HOURLY   
...                      ...                                     ...   
33178  ZYLINSKA,  KATARZYNA                           POLICE OFFICER   
33179     ZYMANTAS,  LAURA C                          POLICE OFFICER   
33180      ZYMANTAS,  MARK E                          POLICE OFFICER   
33181    ZYRKOWSKI,  CARLO E                          POLICE OFFICER   
33182   ZYSKOWSKI,  DARIUSZ                  CHIEF DATA BASE ANALYST   

             Department Full or Part-Time Salary or Hourly  Typical Hours  \
0                POLICE                 F           Salary            NaN   
1                POLICE                 F           Salary            NaN   
2      GENERAL SERVICES                 F           Salary            NaN   
3           WATER MGMNT                 F           Salary            NaN   
4                  OEMC                 P           Hourly           20.0   
...                 ...               ...              ...            ...   
33178            POLICE                 F           Salary            NaN   
33179            POLICE                 F           Salary            NaN   
33180            POLICE                 F           Salary            NaN   
33181            POLICE                 F           Salary            NaN   
33182              DoIT                 F           Salary            NaN   

       Annual Salary  Hourly Rate  
0           101442.0          NaN  
1            94122.0          NaN  
2           101592.0          NaN  
3           110064.0          NaN  
4                NaN        19.86  
...              ...          ...  
33178        72510.0          NaN  
33179        48078.0          NaN  
33180        90024.0          NaN  
33181        93354.0          NaN  
33182       115932.0          NaN  

[33183 rows x 8 columns]

In [33]:
from statsmodels.stats.proportion import proportions_ztest

nobs = salaries["Salary or Hourly"].size
counts = salaries[salaries["Salary or Hourly"]=="Hourly"]["Salary or Hourly"].size
value = 0.25
zstat, pvalue = proportions_ztest( counts, nobs, value)
pvalue

0.0004481127249057967

# APUNTES -----( ESTO SON APUNTES, NO CORREGIR :))

In [20]:

import numpy as np
import pandas as pd
import scipy.stats as stats
import matplotlib.pyplot as plt
import math

In [ ]:
t-test de una muestra

In [ ]:
from scipy.stats import ttest_1samp
import numpy as np
ages = np.genfromtxt(“ages.csv”)
print(ages)
ages_mean = np.mean(ages)
print(ages_mean)
tset, pval = ttest_1samp(ages, 30)
print(“p-values”,pval)
if pval < 0.05:    # alpha value is 0.05 or 5%
   print(" we are rejecting null hypothesis")
else:
  print("we are accepting null hypothesis")

In [ ]:
t-test de dos muestras

In [ ]:
from scipy.stats import ttest_ind
import numpy as np
week1 = np.genfromtxt("week1.csv",  delimiter=",")
week2 = np.genfromtxt("week2.csv",  delimiter=",")

week1_mean = np.mean(week1)
week2_mean = np.mean(week2)
print("week1 mean value:",week1_mean)
print("week2 mean value:",week2_mean)

week1_std = np.std(week1)
week2_std = np.std(week2)
print("week1 std value:",week1_std)
print("week2 std value:",week2_std)

ttest,pval = ttest_ind(week1,week2)
print("p-value",pval)
if pval <0.05:
  print("we reject null hypothesis")
else:
  print("we accept null hypothesis")

In [7]:
from scipy.stats import ttest_ind
import numpy as np


np.random.seed(11)

before= stats.norm.rvs(scale=30, loc=250, size=100)

after = before + stats.norm.rvs(scale=5, loc=-1.25, size=100)

weight_df = pd.DataFrame({"weight_before":before,
                          "weight_after":after,
                          "weight_change":after-before})

weight_df.describe()             # Check a summary of the data

weight_before  weight_after  weight_change
count     100.000000    100.000000     100.000000
mean      250.345546    249.115171      -1.230375
std        28.132539     28.422183       4.783696
min       170.400443    165.913930     -11.495286
25%       230.421042    229.148236      -4.046211
50%       250.830805    251.134089      -1.413463
75%       270.637145    268.927258       1.738673
max       314.700233    316.720357       9.759282


The summary shows that patients lost about 1.23 pounds on average after treatment. Let's conduct a paired t-test to see whether this difference is significant at a 95% confidence level:

In [9]:
ttest,pval=stats.ttest_rel(a = before,
                b = after)
print("p-value",pval)
if pval <0.05:
  print("we reject null hypothesis")
else:
  print("we accept null hypothesis")

p-value 0.011596444318439857
we reject null hypothesis



The p-value in the test output shows that the chances of seeing this large of a difference between samples due to chance is just over 1%, por lo que el experimento si ha causado efecto 

# VARIABLES ALEATORIAS

# 1.- Variables aleatorias discretas o (Probability Mass Function)

In [ ]:
**DISTRIBUCION UNIFORME DISCRETA**

In [ ]:
Es una distribucion de probabilidad que asume un numero finito de valores con la misma probabilidad. 

ej: para un dado , la probabilidad de que al lanzarlo salga 4 es 1/6.Sabemos que un dado tiene 6 caras
ej:Tenemos un anaimal enfermo que sabemos que morirá en cualquier momento en un plazo de 6 meses
¿Cual es la probabilidad de que se muera en 2 meses?

¡creo que se necesita la varianza  para hallarlo??


In [ ]:
**PROCESO DE BERNOULLI**

In [ ]:
Se aplica en aquellas situaciones en las que un cierto atributo se aplica con probabilidad de exito y por tanto sera no existo la ausencia de ese mismo atributo 
Los resultados solo pueden ser dos y son mutuamente excluyentes. aprobado /suspenso. Defectuoso/ no defectuoso 

ej: Lanzar una moneda al aire 
ej: las ventas d eproducccion mensuales s epueden llograr o no lograr 
ej : En pruebas de seleccion multiple aunque halla 4 o 5 alternativas sepuede clasificar como correcta o incorrecta.
ej: Un tratamiento médico puede ser efectivo o inefectivo 
    
    

In [ ]:
**DISTRIBUCION BINOMINIAL**

In [ ]:
Numero de exitos ( cuantos exitos, que % ,que numero se obtiene)
al realizar N observaciones o pruebas. Las variables deben de ser independientes.

ej UN reciente estudio de la asociacion de conductores en autopista, ha revelado que el 60 % de los conductores en autopista
usa el cinturon de seguridad. Se seleciona una muestra de 10 conductores del estado de Oklahoma

¿Cual es la probabilidad de que 7 de los conductores lleven el cinturon?
-Comprobamos que solo hay dos posibles : exito : llevan el cinturon o fracao: no lo llevan 
-La probabilidad de exito para todos los casos es la misma e invariable : 60 %
- Son variables independienes, si ahora se le hace la prueba a un octavo conductor, no condicionará el resultado .
-SE define una variable aleatoria: "numero de conductores que pueden llevar el cinturon ", es decir, numero de exitos 
    
    
    
ej : En una ciudad el 40 % de los votantes está a favor del partido P. Se toma una muestra aleatoria de 10 votantes 
y se observa entre ellos quienes apoyan a P ¿Cual es la probabilidad de que en dicha muestra 6 apoyen el partido?
-Son eventos independientes, la referencia o no de cada uno de los votantes elegidos no depende de los otros votantes

-------DISTRIBUCION BINOMINIAL NEGATIVA sería igual , pero lo que queremos saber no son los exitos, sino cuantas pruebas debemos hacer----
ej.  el número de visitas de ventas que necesita realizar para cerrar 10 pedidos

In [ ]:
**DISTRIBUCION DE POISSON** 

In [ ]:
Describe el número de veces que se produce un evento en un determinado intervalo (normalmente de tiempo)

Ej el número de llamadas telefónicas por minuto,
el número de errores por página en un documento o 
el número de defectos por cada 100 metros de material.

In [ ]:
**DISTRIBUCION GEOMETRICA**
Numero de fracasos obtenidos ( cuantos fracaso, que % ,que numero se obtiene)
al realizar N observaciones o pruebas. Las variables deben de ser independientes.Describe el número de pruebas hasta la primera vez que resulta correcta

ej: Número de veces que gira un ruleta hasta ganar 
o cuántos pozos hay que perforar hasta encontrar petróleo.

ej.Una compañía de seguros desea describir el número de reclamaciones recibidas hasta recibir una reclamación de gran envergadura. Los registros indican que el 6% de las reclamaciones recibidas equivalen en dólares a todo el resto de reclamaciones juntas.

En este ejemplo, el parámetro Probabilidad es un 0,06 (6%) de probabilidad de recepción de una reclamación de envergadura.

# 2.- Variables aleatorias continuas o (Density Mass Function)

In [ ]:
Distribución t de Student (n)

In [ ]:
Distribucion Z

In [ ]:
Distribución ji-cuadrado (n) .

In [ ]:
ANOVA 

# REGRESIONES

# 1.REGRESION LINEAL